stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/RetrofitApi.kt

```diff
@@ -256,7 +256,7 @@ internal interface RetrofitApi {
         @Query("user_id") userId: String,
         @Query("client_id") connectionId: String,
         @Body body: MuteUserRequest
-    ): RetrofitCall<MuteUserResponse>
+    ): RetrofitCall<CompletableResponse>
```

Comment:
Did we check with the backend team that this was an intentional change in the API on their end?

Reply:
I've double-checked that with other SDK and iOS, Flutter and js-client return `EmptyResponse` instead of `Mute` so I assumed that we've been returning `Mute` by mistake